components/navigation.css

```diff
@@ -0,0 +1,105 @@
+.main-header {
+    position: sticky;
+    top: 0;
+    z-index: 999;
+    width: 100vw;
+    background: white;
+    overflow: hidden;
+}
+
+.main-nav,
+.form-search .search-container,
+.nav-search,
+.uu-nav-links,
+.uu-nav-links li {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.main-nav {
+    justify-content: space-around;
+    padding: var(--sp-8);
+    width: 100vw;
+    box-shadow: 0 0 5px 1px var(--gray);
+}
+
+.uu-logo {
+    order: -1;
+    height: 6rem;
+    width: 6rem;
```

Comment:
you can prefer giving max or min-width for responsiveness part later on:)

Reply:
Sure Samyak, thanks for the suggestion.